### PR TITLE
Update culture names list

### PR DESCRIPTION
**Pull request by hishamco**
I create this PR based on issue #98

### CODE REVIEW

File: src/Microsoft.Extensions.Globalization.CultureInfoCache/CultureInfoList.cs [hunk 1]
```diff
@@ -431,6 +820,8 @@ public static partial class CultureInfoCache
             "zh",
             "zh-CN",
             "zh-Hans",
+            "zh-Hans-HK",
```
Comment (Kagamine) [inline src/Microsoft.Extensions.Globalization.CultureInfoCache/CultureInfoList.cs]: It should be zh-Hant-HK & MO, see #104

Comment (hishamco) [inline src/Microsoft.Extensions.Globalization.CultureInfoCache/CultureInfoList.cs]: As I mentioned before in #103 this generated by tool

**Comment by kirthik** [comment]
Updating `CultureInfoList` with just one `CultureInfo` may not be the right solution. Closing this one since we already have another PR - https://github.com/aspnet/Localization/pull/104 and a related issue for it - https://github.com/aspnet/Localization/issues/98.

**Comment by hishamco** [comment]
@kirthik i'm still surprised why this PR has been closed, while it fix issue #98 after @DamianEdwards confirm. Regarding PR #104 i'm not sure if @Kagamine added the cultures manually or using the ```CultureInfoGenerator``` command, furthermore if you look to the updated list may you will understand that more than 400+ cultures have been added

**Comment by hishamco** [comment]
@Eilon is this closed by mistake?!!

**Comment by Kagamine** [comment]
I think she would close the PR #104 ...

**Comment by hishamco** [comment]
That's what I expect :smile:

**Comment by Kagamine** [comment]
Why you care it so much?

**Comment by hishamco** [comment]
Because the PR sent after @DamianEdwards confirmation, furthermore the culture that you come up with is interesting and may we find a smart solution to generate all the culture cross platform
thanks @Kagamine

**Comment by Eilon** [comment]
We will discuss these next week and get back to you.

**Comment by hishamco** [comment]
Thanks @Eilon

**Comment by Eilon** [comment]
@kirthik given our discussion today, I believe this is ok to merge.

**Comment by kirthik** [comment]
@Eilon : Based on the discussion in https://github.com/aspnet/Localization/issues/111, plan is to remove `KnownCultureNames`. Then this PR will not be relevant right?

**Comment by Eilon** [comment]
@kirthik oh yeah I was confusing the cache and the known list.

**Comment by kirthik** [comment]
Closing this PR since it will no longer be relevant after https://github.com/aspnet/Localization/issues/111 is implemented.

**Comment by hishamco** [comment]
@kirthik should issue #98 will be closed too?!!

**Comment by Eilon** [comment]
Yes, I closed #98.

**Pull request closed without merging**